settings.gradle

```diff
@@ -24,5 +24,3 @@ include ':shark-hprof-test'
 include ':shark-log'
 include ':shark-test'
 include ':leakcanary-deobfuscation-gradle-plugin'
-
-enableFeaturePreview("VERSION_CATALOGS")
```

Comment:
The [Version catalogs](https://docs.gradle.org/7.4/userguide/platforms.html) is promoted to a stable feature.